hydro_cli/examples/simple/main.rs

```diff
@@ -1,6 +1,66 @@
-fn main() {
+use std::collections::HashMap;
+
+use hydroflow::{
+    hydroflow_syntax,
+    util::{connection::ConnectionPipe, unix_lines},
+};
+use tokio::net::{UnixListener, UnixStream};
+
+#[tokio::main]
+async fn main() {
     let mut input = String::new();
     std::io::stdin().read_line(&mut input).unwrap();
     let trimmed = input.trim();
-    println!("Hello, world, I got {trimmed}!");
+
+    let connection_pipes =
+        serde_json::from_str::<HashMap<String, ConnectionPipe>>(&trimmed).unwrap();
+
+    if connection_pipes.contains_key("bar") {
```

Comment:
The idea is that this logic will be replaced with some type of macro / runtime utility.